app/models/audit_trail.rb

```diff
@@ -16,21 +16,51 @@ def changed_fields
     relevant_fields = modified.map do |key, _value|
       key unless IRRELEVANT_FIELDS.include? key
     end
-
     relevant_fields.compact.map(&:humanize)
   end
 
   # TODO: properly render null values like in special circumstances
   # Something like this should work:
   # "HAHA I WIN" if (fields.all?{|f| f.blank? || (f.flatten.all? &:blank? if f.respond_to?('each'))})
+
+  def format_dates(hash)
+    for key in ['appointment_date', 'initial_call_date', 'pledge_generated_at']
+      if hash.has_key? key
+        hash[key] = Date.strptime(hash[key].to_s, "%Y-%m-%d %H:%M:%S %Z").strftime('%m/%d/%Y')
```

Comment:
IIRC this stuff is all saved as date objects in the audit trail, so we might be able to get away with `hash[key] = hash[key].display_date` here. I'm good either way though.